fixieai/agents/code_shot.py

```diff
@@ -99,4 +99,7 @@ def _split_few_shots(few_shots: str) -> List[str]:
     few_shot_splits = [few_shot_splits[0]] + [
         "Q:" + few_shot for few_shot in few_shot_splits[1:]
     ]
-    return few_shot_splits
+
+    def not_empty(s):
+        return s != ""
+    return list(filter(not_empty, few_shot_splits))
```

Comment:
The pythonic way to write this, would be:
```python
return [few_shot for few_shot in few_shot_splits if few_shot]
```
List comprehensions are as fast as filter and map, and they are more a bit more readable.